### DISCUSSION TITLE
Meme Coin On BTC with Rune Protocol

### DISCUSSION THREAD

**Discussion by runicswap-team** [community/community#143340] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

👋 Welcome!
We’re using Discussions as a place to connect with other members of our community. We hope that you:

Ask questions you’re wondering about.
Share ideas.
Engage with other community members.
Welcome others and are open-minded. Remember that this is a community we
build together 💪.
To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Let’s make Bitcoin a little more fun!

🚀 We’re opening a discussion about creating a meme coin on the BTC chain. Drop by our GitHub Community to join the conversation and be part of the journey from day one!

**Comment by runicswap-team** [comment]
Could you share your idea in detail?

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/143341#discussioncomment-11125241) for reference to why this discussion was closed and locked. Thanks!
